.ci/pitest.sh

```diff
@@ -228,7 +228,7 @@ pitest-coding-require-this-check)
 --list)
   echo "Supported profiles:"
   pomXmlPath="$(dirname "${0}")/../pom.xml"
-  sed -n -e 's/<id>\(pitest-\w\+\)<\/id>/\1/p' < "${pomXmlPath}" | sort
+  sed -n -e 's/<id>\(pitest-.\+\)<\/id>/\1/p' < "${pomXmlPath}" | sort
```

Comment:
Why is it `\+` and not just `+`?

Reply:
We need not greedy any symbol.
`.*?`
https://www.ultraedit.com/support/tutorials-power-tips/ultraedit/non-greedy-perl-regex.html

Reply:
> Why is it `\+` and not just `+`?

https://www.gnu.org/software/sed/manual/html_node/Regular-Expressions.html

It mentions `\+`

Reply:
> We need not greedy any symbol. `.*?` https://www.ultraedit.com/support/tutorials-power-tips/ultraedit/non-greedy-perl-regex.html

Good suggestion but `sed` doesn't support non-greedy quantifiers, so I used `perl`, not a `perl` expert, please share if it can be simplified.